test/hotspot/jtreg/gc/stringdedup/TestStringDeduplicationTools.java

```diff
@@ -63,6 +65,12 @@
         }
     }
 
+    public static void maybeOverrideGC(String[] args) {
+        if (args.length > 0) {
+            overridingGC = args[0];
+        }
+    }
+
```

Comment:
I think it would be good to parse the gc string. Otherwise it might be easy to make a spelling mistake or add another argument (before) to the driver without realizing the error.

Reply:
I think we should just blindly pass this option (or options) to the spawned JVMs, without looking at them.

Reply:
I don't think there's any point to doing any parsing here, and good reason not to.  If the value isn't sensible as a GC name the test will very likely fail because of an invalid CLA to the subprocess.  And validating that it's a valid GC name involves using vm.gc.GC, which brings in WhiteBox and all its attendant cruft.

Reply:
Okay, but is it not better then to skip the string concatenation part and give the complete flag directly as an argument to the driver directive?